x/poolmanager/router.go

```diff
@@ -362,12 +365,14 @@ func (k Keeper) RouteExactAmountOut(ctx sdk.Context,
 			return sdk.Int{}, poolErr
 		}
 
-		// check if pool is active, if not error
+		// check if pool has swaps enabled
```

Comment:
I think the old one is correct because inactive pool also means that LP logic is disabled
```suggestion
		// check if pool is active, if not error
```